packages/confirmation-modal/src/ConfirmationModal.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@leafygreen-ui/emotion';
+import Button, { Variant } from '@leafygreen-ui/button';
+import Modal from '@leafygreen-ui/modal';
+import { uiColors } from '@leafygreen-ui/palette';
+import TextInput from '@leafygreen-ui/text-input';
+
+const titleStyle = css`
+  color: ${uiColors.gray.dark2};
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 25px;
+
+  margin-bottom: 10px;
+`;
+
+const baseModalStyle = css`
+  width: 600px;
+  padding: initial;
+`;
+
+const contentStyle = css`
+  color: ${uiColors.gray.dark1};
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 24px;
+
+  padding: 36px;
+`;
+
+const textEntryInputStyle = css`
+  width: 300px;
+  padding-top: 14px;
+`;
+
+const footerStyle = css`
+  &:before {
```

Comment:
Is there a reason we need to use a separate div for the pseudo-element for this border? From what I see, we should be able to combine these rules with the below `footerContentStyle` and remove unnecessary nesting of divs within the footer content.

Reply:
I made it its own div so that the border wouldn't be affected by the margin of `footerContentStyle`. If I merged them then I think I would need to replace the margin-top of the footer content with a margin/padding-bottom for the pseudo-element which is less clean.

Reply:
I played a bit around with this, and it looks like you can simply move the `border-top: ${uiColors.gray.light2};` to `footerContentStyle`, and remove this div altogether, pseudo-element and all. It achieves exactly the same effect with one less element in the DOM, and quite a bit less CSS.

Reply:
<img width="1569" alt="Screen Shot 2020-08-10 at 4 10 34 PM" src="https://user-images.githubusercontent.com/25213299/89826315-11abd180-db24-11ea-9392-b62ab85d8556.png">